Description: - Adding the Shebang Line in .py files.
Author: Giovani Augusto Ferreira <giovani@debian.org>
Last-Update: 2017-07-30
Index: pompem-0.2.0/common/__init__.py
===================================================================
--- /dev/null
+++ pompem-0.2.0/common/__init__.py
@@ -0,0 +1 @@
+#!/usr/bin/python3
Index: pompem-0.2.0/common/print_messages.py
===================================================================
--- pompem-0.2.0.orig/common/print_messages.py
+++ pompem-0.2.0/common/print_messages.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
 HELP_MESSAGE = """
Index: pompem-0.2.0/common/writers.py
===================================================================
--- pompem-0.2.0.orig/common/writers.py
+++ pompem-0.2.0/common/writers.py
@@ -1,3 +1,4 @@
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 import os
 import sys
Index: pompem-0.2.0/core/__init__.py
===================================================================
--- /dev/null
+++ pompem-0.2.0/core/__init__.py
@@ -0,0 +1 @@
+#!/usr/bin/python3
Index: pompem-0.2.0/core/exploit_finder.py
===================================================================
--- pompem-0.2.0.orig/core/exploit_finder.py
+++ pompem-0.2.0/core/exploit_finder.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
 NUM_WORKERS = 5
Index: pompem-0.2.0/core/request_worker.py
===================================================================
--- pompem-0.2.0.orig/core/request_worker.py
+++ pompem-0.2.0/core/request_worker.py
@@ -1,3 +1,4 @@
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 from threading import Thread
 import requests
Index: pompem-0.2.0/core/scrapers.py
===================================================================
--- pompem-0.2.0.orig/core/scrapers.py
+++ pompem-0.2.0/core/scrapers.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
 from threading import Thread
Index: pompem-0.2.0/pompem.py
===================================================================
--- pompem-0.2.0.orig/pompem.py
+++ pompem-0.2.0/pompem.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
 import optparse
